sympy/geometry/ellipse.py

```diff
@@ -1339,9 +1341,16 @@ class Circle(Ellipse):
     >>> c2.hradius, c2.vradius, c2.radius, c2.center
     (sqrt(2)/2, sqrt(2)/2, sqrt(2)/2, Point2D(1/2, 1/2))
 
+    >>> # a circle object is returned
+    >>> from sympy.abc import x, y, a, b
+    >>> Circle(equation = x ** 2 + y ** 2 - 25)
+    Circle(Point2D(0, 0), 5)
+    >>> Circle(equation = a ** 2 + b ** 2 - 25, x='a', y='b')
+    Circle(Point2D(0, 0), 5)
+
     """
 
-    def __new__(cls, *args, **kwargs):
+    def __new__(cls, *args, equation=None, x='x', y='y', **kwargs):
```

Comment:
In [Python 2.7](https://docs.python.org/2.7/reference/compound_stmts.html#function-definitions), all named parameters (possibly with default value) must come before `*args` and `**kwargs`.

Reply:
@jksuom  I've fixed that now. Please take a look.